IMPLEMENTATION_COVERAGE.md

```diff
@@ -751,12 +751,11 @@
 
 ## bedrock
 <details>
-<summary>39% implemented</summary>
+<summary>43% implemented</summary>
 
 - [ ] create_evaluation_job
 - [ ] create_guardrail
 - [ ] create_guardrail_version
-- [ ] create_model_copy_job
```

Comment:
Usually when the `implementation`-script suddenly removes methods like this, it's because the installed `botocore` module is outdated, and the feature didn't exist yet in the older `botocore` module.

We'll always update the docs before an actual release, so it's not a big deal if these methods disappear inbetween, just as an FYI; if you update `botocore`/`boto3` before running the `implementation`-script, this shouldn't happen.